snap/quota/resources.go

```diff
@@ -203,14 +203,19 @@ func (qr *Resources) CheckFeatureRequirements() error {
 	return nil
 }
 
+// Unset returns true if there is no limit set
+func (qr *Resources) Unset() bool {
+	return qr.Memory == nil && qr.CPU == nil && qr.CPUSet == nil && qr.Threads == nil && qr.Journal == nil
```

Comment:
Would
```suggestion
	return *qr == Resources{}
```
work as well?